src/components/DeveloperBlog.tsx

```diff
@@ -58,13 +58,23 @@ const DeveloperBlog = () => {
     <>
       <Grid container spacing={2}>
         {blogItems.map((post, index) => {
+          let thumbnail_url = post.thumbnail;
+          try {
+            const thumbnail_host = new URL(post.thumbnail).host;
+            if (thumbnail_host.endsWith('ucarecdn.com')) {
+              thumbnail_url = `${thumbnail_url}-/resize/800x/-/format/auto/-/quality/lightest/`;
+            }
+          } catch {
+            thumbnail_url = `https://blog.geolonia.com${thumbnail_url}`;
+          }
```

Comment:
最初に `thumbnail_url` に `post.thumbnail` を代入し、それを後で使いまわす形だと、現在の `thumbnail_url` の値が何なのか分かりづらくなり、混乱しやすくなるかと思います。(私はしました)
以下のように `thumbnail_url` には最初は何も代入せず代入の回数が1回になるようにし、また、最初に代入した `thumbnail_url` の値を使う場合は、直接 `post.thumbnail` を使用する形でどうでしょうか?

後、変数の命名規則について、他の場所は camelCase のようなので、snake_case から変更してあります。snake_case の方が望ましければ、お手数ですが戻して下さい。

```suggestion
          let thumbnailUrl;
          try {
            const thumbnailHost = new URL(post.thumbnail).host;
            if (thumbnailHost.endsWith('ucarecdn.com')) {
              thumbnailUrl = `${post.thumbnail}-/resize/800x/-/format/auto/-/quality/lightest/`;
            } else {
              thumbnailUrl = post.thumbnail;
            }
          } catch {
            thumbnailUrl = `https://blog.geolonia.com${post.thumbnail}`;
          }
```

Reply:
ありがとうございます、おっしゃる通りだと思います 🙇